Doc/library/inspect.rst

```diff
@@ -765,6 +765,18 @@ function.
        .. versionadded:: 3.10
           ``globalns`` and ``localns`` parameters.
 
+   .. classmethod:: Signature.from_code(co)
+
+       Return a :class:`Signature` (or its subclass) object
+       for a given :class:`code object <types.CodeType>` ``co``.
+
+       Since code objects do not know anything
+       about default values or annotations,
+       they will always be omitted from the signature.
```

Comment:
Lead with the important thing, then follow up with the explanation:

```suggestion
       Default values and annotations
       will not be included in the signature,
       since code objects have no knowledge of such things.
```